plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { SelectControl } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { Link, useFormContext } from '@woocommerce/components';
+import {
+	EXPERIMENTAL_PRODUCT_SHIPPING_CLASSES_STORE_NAME,
+	Product,
+	ProductShippingClass,
+} from '@woocommerce/data';
+import interpolateComponents from '@automattic/interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import { ProductSectionLayout } from '../layout/product-section-layout';
+import { getTextControlProps } from './utils';
+import { ADMIN_URL } from '../../utils/admin-settings';
+
+const DEFAULT_SHIPPING_CLASS_OPTIONS: SelectControl.Option[] = [
+	{ value: '', label: __( 'No shipping class', 'woocommerce' ) },
+	{ value: '-1', label: __( 'Standard shipping', 'woocommerce' ) },
```

Comment:
Thanks for leaving a question surrounding this in the original issue :)

Reply:
According to this [comment](https://github.com/woocommerce/woocommerce/issues/34328#issuecomment-1249083187) we will have only the 'No shipping class' option and it will be the default.